addons/storysource/package.json

```diff
@@ -25,7 +25,7 @@
     "babel-runtime": "^6.26.0",
     "estraverse": "^4.2.0",
     "loader-utils": "^1.1.0",
-    "prettier": "~1.12.1",
+    "prettier": "^1.12.1",
```

Comment:
Please update it to latest version: looks like we don't support `1.12` after change in parser imports